pkg/koordlet/metricsadvisor/collectors/coldmemoryresource/cold_page_kidled_test.go

```diff
@@ -38,6 +39,120 @@ import (
 	mock_statesinformer "github.com/koordinator-sh/koordinator/pkg/koordlet/statesinformer/mockstatesinformer"
 )
 
+func Test_kideldEnable(t *testing.T) {
+	type fields struct {
+		SetSysUtil func(helper *system.FileTestUtil)
+		fg         map[string]bool
+	}
+	tests := []struct {
+		name        string
+		fields      fields
+		wantsupport bool
+		wantenable  bool
+	}{
+		{
+			name: "os doesn't support kidled and koordlet feature-gate doesn't support kidled",
+			fields: fields{
+				SetSysUtil: func(helper *system.FileTestUtil) {
+					system.Conf.SysRootDir = filepath.Join(helper.TempDir, system.Conf.SysRootDir)
```

Comment:
the `system.FileTestUtil` will initialize the SysRootDir by default, so we do not need to adjust it